### PR TITLE
Add graphql autogen into codegen script

### PR DESCRIPTION
**Pull request by edwardfeng-db**
<details><summary>&#x1F6E0 DevTools &#x1F6E0</summary>
<p>

[Open in GitHub Codespaces](https://codespaces.new/edwardfeng-db/mlflow/pull/10850?quickstart=1)

#### Install mlflow from this PR

```
pip install git+https://github.com/mlflow/mlflow.git@refs/pull/10850/merge
```

#### Checkout with GitHub CLI

```
gh pr checkout 10850
```

</p>
</details>

### Related Issues/PRs

 #xxx

https://github.com/mlflow/mlflow/pull/10474

### What changes are proposed in this pull request?

- Add graphql codegen script into `./dev/generate-protos.sh`
- Add readme for graphql codegen

### How is this PR tested?

- [ ] Existing unit/integration tests
- [ ] New unit/integration tests
- [ ] Manual tests

### Does this PR require documentation update?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. I've updated:
  - [ ] Examples
  - [ ] API references
  - [ ] Instructions

#### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

#### What component(s), interfaces, languages, and integrations does this PR affect?

Components

- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/deployments`: MLflow Deployments client APIs, server, and third-party Deployments integrations
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/recipes`: Recipes, Recipe APIs, Recipe configs, Recipe Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface

- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language

- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations

- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>

#### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: dev/generate-protos.sh [hunk 1]
```diff
@@ -59,3 +59,6 @@ rm "$PROTOS/internal_pb2.py.old"
 rm "$PROTOS/databricks_uc_registry_messages_pb2.py.old"
 rm "$PROTOS/databricks_uc_registry_service_pb2.py.old"
 rm "$TEST_PROTOS/test_message_pb2.py.old"
+
+python3 ./dev/proto_to_graphql/code_generator.py
```
Comment (harupy) [inline dev/generate-protos.sh:63]:
```suggestion
python ./dev/proto_to_graphql/code_generator.py
```

File: requirements/test-requirements.txt [hunk 2]
```diff
@@ -45,3 +45,4 @@ openai<1.0
 # Required for showing pytest stats
 psutil
 graphene<4
+google
```
[the remark this answers is not in the document]
Comment (edwardfeng-db) [reply requirements/test-requirements.txt:48]: I realized that the `protos` github action is not installing the proper dependencies. But yeah this is not correct. So far it only installs protoc, but we'll have to make it install many other deps. Curious if you have idea on how to do it. Seems like right now it's just doing wget but if we do this we'll need to write a lot of complicated wgets

[the remark this answers is not in the document]
Comment (WeichenXu123) [reply requirements/test-requirements.txt:48]: You can add installation command inside specific github action step.

**Comment by harupy** [comment]
> @harupy I've decided to only add the README for now as I realized that the script dependency thing could be tricky to do because we need `mlflow` itself, which doesn't contain the graphql stuff right now. I think we might just do that part after we merge the feature branch and I'll directly commmit those into master

sounds good!

**Review by harupy — APPROVED** [review]

**Pull request merged**